test-framework/core/src/main/java/org/keycloak/test/framework/events/SysLogServerSupplier.java

```diff
@@ -49,4 +51,22 @@ public void close(InstanceContext<SysLogServer, InjectSysLogServer> instanceCont
     public boolean compatible(InstanceContext<SysLogServer, InjectSysLogServer> a, RequestedInstance<SysLogServer, InjectSysLogServer> b) {
         return true;
     }
+
+    @Override
+    public KeycloakServerConfigBuilder intercept(KeycloakServerConfigBuilder serverConfig, InstanceContext<SysLogServer, InjectSysLogServer> instanceContext) {
```

Comment:
Does it mean that syslog listener is enabled permanently?

Reply:
Nopes, only requested or deployed suppliers intercept

Reply:
SysLogServer is using global scope though, so once it's started it will keep running, and will configure the realms; that's a tradeoff as otherwise we'd stop/start the test every time a test needs events and one doesn't.